Object Storage/request-rate-and-performance-considerations.md

```diff
@@ -24,4 +24,4 @@ PUT-intensive workloads, such as Backup to cloud methodologies, should be segmen
 **Quick Tip:** each user must have a unique email address and while this email is not used for any service delivery, clients may need to use a convention that allows reuse of the same email address. To do so use a ```+``` approach such as first.lastname+obj1@ctl.io, first.lastname+obj2@ctl.io, first.lastname+obj3@ctl.io.
 
 ##### Workloads with a Mix of Request Types
-Clients with a wide range of request types may have to use market benchmark tools (during staging) and user performance monitoring (during production) to evaluate performance levels real-time and align them with business service levels. In the end the same platform maximums are imposed on a per access key ID (user) so care should be taken to scale users and buckets or augment web based content with [CDN services](http://www.level3.com/en/products/content-delivery-network/).
+Clients with a wide range of request types may have to use market benchmark tools (during staging) and user performance monitoring (during production) to evaluate performance levels real-time and align them with business service levels. In the end the same platform maximums are imposed on a per access key ID (user) so care should be taken to scale users and buckets or augment web based content.
```

Comment:
why are we removing level3's CDN options? isn't that product still for sale? just the CLC CDN is not a thing anymore.